options/options_base.hpp

```diff
@@ -20,9 +20,14 @@
 #include <include/optional.hpp>
 #include <include/boost_asio_beast.hpp>
 #include <string>
+#include <map>
 
 namespace bzn
 {
+    // Suffixes for the max size parser.
+    const std::map<char, size_t> BYTE_SUFFIXES{{'B', 1}, {'K', 1024}, {'M', 1048576}, {'G', 1073741824}, {'T', 1099511627776}};
```

Comment:
I would think about moving this into options_base and making it public.

Reply:
Cool. Moved BYTE_SUFFIXES into options_base, and made it static. Also propagated the change through the code.